### PR TITLE
Manual feedback calculation notebook

### PR DESCRIPTION
**Pull request by r-ford**
This notebook currently just loads in CMIP6 data and has a few example plots.

### CODE REVIEW

**Comment by r-ford** [comment]
The notebook: https://projectpythia.org/radiative-feedback-cookbook/_preview/5/notebooks/feedback-analysis/manual-calc

File: notebooks/feedback-analysis/manual-calc.ipynb [hunk 1]
```diff
@@ -34,37 +26,22 @@
...
-    "Following your overview, tell your reader what concepts, packages, or other background information they'll **need** before learning your material. Tie this explicitly with links to other pages here in Foundations or to relevant external resources. Remove this body text, then populate the Markdown table, denoted in this cell with `|` vertical brackets, below, and fill out the information following. In this table, lay out prerequisite concepts by explicitly linking to other Foundations material or external resources, or describe generally helpful concepts.\n",
-    "\n",
-    "Label the importance of each concept explicitly as **helpful/necessary**.\n",
     "\n",
     "| Concepts | Importance | Notes |\n",
     "| --- | --- | --- |\n",
-    "| [Intro to Cartopy](https://foundations.projectpythia.org/core/cartopy/cartopy.html) | Necessary | |\n",
-    "| [Understanding of NetCDF](https://foundations.projectpythia.org/core/data-formats/netcdf-cf.html) | Helpful | Familiarity with metadata structure |\n",
-    "| Project management | Helpful | |\n",
+    "| Loading CMIP6 data | Necessary | |\n",
```
Comment (anacmontoya) [inline notebooks/feedback-analysis/manual-calc.ipynb:41]:
```suggestion
    "| [Loading CMIP6 data](https://projectpythia.org/cmip6-cookbook/notebooks/foundations/intake-esm.html) | Necessary | |\n",
```

**Review by anacmontoya — COMMENTED** [review]
Added link to prerequisites table

File: notebooks/feedback-analysis/manual-calc.ipynb (hunk 1 above)
Comment (r-ford) [reply notebooks/feedback-analysis/manual-calc.ipynb]: Thanks, added in latest commit

**Comment by r-ford** [comment]
To do before merging:
- Temperature
- WV
- CRE
- Figure out ERF for cloud adjustment
- Cloud feedbacks

**Comment by r-ford** [comment]
Merging this unfinished. I'll open an issue outlining the sections that need work.

**Pull request merged**